docs/solution-ideas/articles/dev-test-paas.md

```diff
@@ -39,7 +39,7 @@ This solution is built on the Azure managed services: [Azure DevOps](https://azu
 
 ## Next steps
 
-* [Set up Azure DevOps](https://www.visualstudio.com/docs/setup-admin/get-started)
+* [Set up Azure DevOps](https://docs.microsoft.com/en-us/azure/devops/get-started/?view=azure-devops)
```

Comment:
consider: without the en-us and trailing content  <https://docs.microsoft.com/azure/devops/get-started>